editor/templates/place/edit.html

```diff
@@ -5,6 +5,7 @@
 {%- block styles %}
 {{ super() }}
 <link rel="stylesheet" href="https://unpkg.com/leaflet@1.3.1/dist/leaflet.css" integrity="sha512-Rksm5RenBEKSKFjgI3a41vrjkw4EVPlJ3+OiI65vTjIdo9brlAacEuKOiQ5OFh7cOI1bkDwLqdLw3Zg0cRJAAQ==" crossorigin=""/>
+<link rel="stylesheet" href="https://unpkg.com/leaflet-draw/dist/leaflet.draw.css" />
```

Comment:
Should this have an integrity check like main Leaflet import?

Reply:
Yep. If you add `?meta` to the unpkg URL, it'll give you the integrity check string to add:

```
sha384-NZLkVuBRMEeB4VeZz27WwTRvlhec30biQ8Xx7zG7JJnkvEKRg5qi6BNbEXo9ydwv
```

Reply:
Thanks! 

Resolved via https://github.com/iandees/wof-editor/pull/85/commits/65cd7b0ed6213e4747b1a66916d62f42c7d8cd36.